conf/compaction/rgw/rgw_multisite-5node-1client.yaml

```diff
@@ -0,0 +1,85 @@
+# System Under Test environment configuration for RGW multi site suites.
+globals:
+  - ceph-cluster:
+      name: ceph-pri
+
+      node1:
+        role:
+          - _admin
+          - installer
+          - mgr
+          - mon
+
+      node2:
+        disk-size: 20
+        no-of-volumes: 4
+        role:
+          - mgr
```

Comment:
need to add role rgw for node 2,3 and 4 in both cluster

Reply:
done